geo/pygadm/item/wrapper.py

```
@@ -0,0 +1,30 @@
+__author__ = "Ivan Ruiz Manuel"
+__copyright__ = "Copyright 2025, Ivan Ruiz Manuel"
+__email__ = "i.ruizmanuel@tudelft.nl"
+__license__ = "MIT"
+
+from pathlib import Path
+
+from pygadm import Items
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)